src/requests/process-controller.ts

```diff
@@ -57,16 +58,19 @@ export function processController(methods: ControllerMethod[], name: string, con
     content += '\n';
   }
 
-  content += `@Injectable()\n`;
-  content += `export class ${name}Service {\n`;
-  content += indent('constructor(private http: HttpClient) {}');
-  content += '\n';
+  content += `@Injectable()
+  export class ${name}Service {
```

Comment:
Please, do not use multi-row strings, since the indentation is not under control. You can use `indent` helper function and pass lines in an array, e.g. Check other usages.